tabbycat/tournaments/templates/round_advance_check.html

```diff
@@ -1,26 +1,25 @@
 {% extends "base.html" %}
-{% load debate_tags %}
+{% load i18n debate_tags %}
 
-{% block page-title %}Confirm Round Advance{% endblock %}
+{% block page-title %}{% trans "Confirm Round Advance" as page_title %}{% endblock %}
```

Comment:
If you use the `{% trans string as name %}` syntax, the text actually doesn't render—instead, it just saves it as a local variable. So you'll find on this page the title is blank.